sycl/source/detail/queue_impl.hpp

```diff
@@ -33,6 +33,13 @@ using DeviceImplPtr = shared_ptr_class<detail::device_impl>;
 /// Sets max number of queues supported by FPGA RT.
 const size_t MaxNumQueues = 256;
 
+//// Possible CUDA context types supported by PI CUDA backend
+/// TODO: Implement this as a property once there is an extension document
+enum class cuda_context_type : char { primary, custom };
+
+/// Default context type created for CUDA backend
+const cuda_context_type DefaultContextType = cuda_context_type::custom;
```

Comment:
```suggestion
constexpr cuda_context_type DefaultContextType = cuda_context_type::custom;
```

Reply:
This suggestion has been applied to the commit. Thanks!